_docs/tasks/traffic-management-v1alpha3/ingress.md

```diff
@@ -0,0 +1,579 @@
+   ```
+
+   ```bash
+   export INGRESS_HOST=169.47.243.100:31486
+   export SECURE_INGRESS_HOST=169.47.243.100:32254
+   ```
+
+1. Access the httpbin service with either HTTP or HTTPS using _curl_:
+
+   ```bash
+   curl -I http://$INGRESS_HOST/status/200
+   ```
+
+   ```
+   HTTP/1.1 200 OK
+   server: envoy
+   date: Mon, 29 Jan 2018 04:45:49 GMT
+   content-type: text/html; charset=utf-8
+   access-control-allow-origin: *
+   access-control-allow-credentials: true
+   content-length: 0
+   x-envoy-upstream-service-time: 48
+   ```
+
+   ```bash
+   curl -I -k https://$SECURE_INGRESS_HOST/status/200
+   ```
+
+   ```
+   HTTP/1.1 200 OK
+   server: envoy
+   date: Mon, 29 Jan 2018 04:45:49 GMT
+   content-type: text/html; charset=utf-8
+   access-control-allow-origin: *
+   access-control-allow-credentials: true
+   content-length: 0
+   x-envoy-upstream-service-time: 96
+   ```
+
+1. Access any other URL that has not been explicitly exposed. You should
+   see a HTTP 404 error:
+
+   ```bash
+   curl -I http://$INGRESS_HOST/headers
+   ```
+
+   ```
+   HTTP/1.1 404 Not Found
+   date: Mon, 29 Jan 2018 04:45:49 GMT
+   server: envoy
+   content-length: 0
+   ```
+
+   ```bash
+   curl -I https://$SECURE_INGRESS_HOST/headers
+   ```
+
+   ```
+   HTTP/1.1 404 Not Found
+   date: Mon, 29 Jan 2018 04:45:49 GMT
+   server: envoy
+   content-length: 0
+   ```
+
+## Configuring ingress using a Kubernetes Ingress resource
+
+An Istio Ingress specification is based on the standard [Kubernetes Ingress Resource](https://kubernetes.io/docs/concepts/services-networking/ingress/)
+specification, with the following differences:
+
+1. Istio Ingress specification contains a `kubernetes.io/ingress.class: istio` annotation.
+
+2. All other annotations are ignored.
+
+The following are known limitations of Istio Ingress:
+
+1. Regular expressions in paths are not supported.
+2. Fault injection at the Ingress is not supported.
+
+The `servicePort` field in the Ingress specification can take a port number
+(integer) or a name. The port name must follow the Istio port naming
+conventions (e.g., `grpc-*`, `http2-*`, `http-*`, etc.) in order to
+function properly. The name used must match the port name in the backend
+service declaration.
+
+### Configuring simple Ingress
+
+1. Create a basic Ingress specification for the httpbin service
+
+   ```bash
+   cat <<EOF | kubectl create -f -
+   apiVersion: extensions/v1beta1
+   kind: Ingress
+   metadata:
+     name: simple-ingress
+     annotations:
+       kubernetes.io/ingress.class: istio
+   spec:
+     rules:
+     - http:
+         paths:
+         - path: /status/.*
+           backend:
+             serviceName: httpbin
+             servicePort: 8000
+         - path: /delay/.*
+           backend:
+             serviceName: httpbin
+             servicePort: 8000
+   EOF
+   ```
+ 
+   `/.*` is a special Istio notation that is used to indicate a prefix
```

Comment:
We don't need any of this. Lets just say that the path must follow the ecmascript format.